### DISCUSSION TITLE
[Question] Is there anyway to clear message in Noice buffer

### DISCUSSION THREAD

**Discussion by tuanbass** [folke/noice.nvim#580] [Q&A]
Sometime, especially in debugging, lots of text is printed out.
Is there any way to clean all message in Noice windows? 

Thanks,

**Comment by ogios** [comment]
`:Noice dismiss` ?

**Reply by tuanbass** [reply]
AFAIK, `Noice dismiss` is used to dismiss all Noice popup. 
Sorry if I made any confuse, but I would like to clear all text in Noice Buffer
![image](https://github.com/folke/noice.nvim/assets/9334023/3065b879-6c58-4ca0-a9d3-ea44b3ee9859)
